packages/react-native-reanimated/src/layoutReanimation/web/componentUtils.ts

```diff
@@ -1,8 +1,9 @@
 'use strict';
-import { logger } from '../../common';
+
 import { LayoutAnimationType, ReduceMotion } from '../../commonTypes';
 import type { EasingFunctionFactory } from '../../Easing';
 import { EasingNameSymbol } from '../../Easing';
+import { logger } from '../../logger';
```

Comment:
```suggestion
import { logger } from '../../common';
```

I believe this was imported correctly